contrib/monitored_resource_util/src/main/java/io/opencensus/contrib/monitoredresource/util/ResourceKeyConstants.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.monitoredresource.util;
+
+/**
+ * Constants for collecting resource information.
+ *
+ * @since 0.18
+ */
+public final class ResourceKeyConstants {
```

Comment:
Those don't seem to match the keys defined [here](https://github.com/census-ecosystem/opencensus-go-resource/blob/master/resourcekeys/const.go).
We must be consistent across all languages for the well-known environments.

Reply:
+1

Reply:
Done.

> We must be consistent across all languages for the well-known environments.

Agreed. We should define these keys in somewhere in the specs same as : https://github.com/census-instrumentation/opencensus-specs/blob/master/utils/MonitoredResource.md#supported-monitored-resources